pkgs/test/lib/src/runner/browser/dom.dart

```diff
@@ -2,100 +2,74 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-// ignore: deprecated_member_use
-import 'dart:js_util' as js_util;
+import 'dart:js_interop';
+import 'dart:js_interop_unsafe';
 
 // ignore: deprecated_member_use
-import 'package:js/js.dart';
-
-@JS()
-@staticInterop
-class Window extends EventTarget {}
+import 'dart:js_util' as js_util;
```

Comment:
This remains because there's a `callConstructor` call below and I'm not sure what the `js_interop` equivalent of it is. @srujzs could you advise please?